source/common/http/http2/codec_impl.cc

```diff
@@ -964,6 +969,7 @@ int ConnectionImpl::onInvalidFrame(int32_t stream_id, int error_code) {
       return 0;
     }
   }
+  }
```

Comment:
indentation seems a bit odd.  Could you remove the extra `{}`'s after `case NGHTTP2_ERR_HTTP_MESSAGING:`